ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/tls/DecryptionParameters.scala

```diff
@@ -39,12 +42,20 @@ case class DecryptionParameters(
     transformation.split("/")(0)
   }
 
+  // According to MIME's section of java.util.Base64 javadoc "All line separators or other
+  // characters not found in the base64 alphabet table are ignored in decoding operation."
+  // For this reason a buffer needs to be screened whether it contains only the allowed
+  // Base64 characters before attempting to decode it.
   private def decodeBase64OrGetVerbatim(encrypted: Array[Byte]): Array[Byte] = {
-    val potentialBase64Char =
+    val allowedBase64Char =
       "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz/+=\n\r".getBytes()
```

Comment:
We pass UTF-8 charset explicitly to getBytes method to emphasize that we care about 8 bit ascii compatible encoding